cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModel.kt

```diff
@@ -83,7 +84,8 @@ data class MessageMetaData(
 @Serializable
 data class Campaigns(
     @SerialName("CCPA") val ccpa: CCPA?,
-    @SerialName("GDPR") val gdpr: GDPR?
+    @SerialName("GDPR") val gdpr: GDPR?,
+    @SerialName("usnat") val usNat: USNatConsentData?,
```

Comment:
so are we going to use single entity for all the `UsNat` responses? if so, that is beautiful

Reply:
yes